src/dsp_tools/commands/xmlupload/resource_create_client.py

```diff
@@ -141,6 +142,9 @@ def make_values(p: XMLProperty) -> list[dict[str, Any]]:
                     last_prop_name = int_prop_name
                 case _:
                     properties_serialised.update({prop_name(prop): make_values(prop)})
+        if resource.iiif_uri:
+            properties_graph += _make_iiif_uri_value(resource.iiif_uri, res_bnode, self.permissions_lookup)
+            last_prop_name = KNORA_API.hasStillImageFileValue
```

Comment:
What is `last_prop_name`? I don't understand what it is doing. Could you perhaps write a comment into the code?

Reply:
I added a comment does it make sense now?